default/main.py

```diff
@@ -1,8 +1,10 @@
 import re
-from fastapi import FastAPI, Request, Form, HTTPException
+import requests
```

Comment:
One of the reasons we're using `FastAPI` is because it's async and more efficient underneath than Flask. `requests` isn't async either. I'd suggest using https://docs.aiohttp.org/en/stable/